tests/languages/node/test.js

```diff
@@ -65,18 +65,18 @@ async function start() {
     response = await general.redirect();
     console.log(response.result);
 
-    response = await general.upload('string', 123, ['string in array'], InputFile.fromPath(__dirname + '/../../resources/file.png', 'file.png'));
+    response = await general.upload('string', 123, ['string in array'], Payload.fromFile(__dirname + '/../../resources/file.png', 'file.png').toFile());
     console.log(response.result);
 
-    response = await general.upload('string', 123, ['string in array'], InputFile.fromPath(__dirname + '/../../resources/large_file.mp4', 'large_file.mp4'));
+    response = await general.upload('string', 123, ['string in array'], Payload.fromFile(__dirname + '/../../resources/large_file.mp4', 'large_file.mp4').toFile());
```

Comment:
We shouldn't need toFile here
fromFile and toFile seems bad dx

Reply:
Agreed. The toFile() conversion should happen inside the createFile() endpoint